examples/flagsmith-on-eks/rds.tf

```diff
@@ -0,0 +1,56 @@
+data "aws_vpc" "cluster" {
+  id = data.aws_eks_cluster.cluster.vpc_config[0].vpc_id
+}
+
+resource "aws_db_subnet_group" "rds" {
+  name       = "rds"
+  subnet_ids = data.aws_eks_cluster.cluster.vpc_config[0].subnet_ids
+}
+
+resource "aws_security_group" "rds" {
+  name   = "flagsmith_rds"
+  vpc_id = data.aws_vpc.cluster.id
+
+  ingress {
+    from_port   = 5432
+    to_port     = 5432
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 5432
+    to_port     = 5432
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = {
+    Name = "flagsmith_rds"
+  }
+}
+
+resource "aws_db_parameter_group" "flagsmith_db" {
+  name   = "flagsmith"
+  family = "postgres11"
+
+  parameter {
+    name  = "log_connections"
+    value = "1"
+  }
+}
+
+resource "aws_db_instance" "flagsmith_db" {
+  identifier             = "flagsmith-db"
+  instance_class         = "db.t3.micro"
+  allocated_storage      = 5
+  db_name                = "flagsmith"
+  engine                 = "postgres"
+  engine_version         = "11.12"
```

Comment:
What do you think about bumping this to 12? We have an issue [here](https://github.com/Flagsmith/flagsmith/issues/1891) to do that for our own RDS instance - otherwise we're providing defaults to customers that will need to be upgraded sooner rather than later.

Reply:
I'm down for it. FYI I used self-hosted Compose file and Helm charts as a reference for this.